cmd/cmd.js

```diff
@@ -264,12 +264,21 @@ class Cmd {
   test() {
     program
       .command('test [file]')
-      .option('-n , --node [node]', __('Node to connect to (default: vm)'))
+      .option('-n , --node <node>', __('Node to connect to. Valid values are ["vm", "embark", "<custom node endpoint>"]: \n') + 
+              '                       vm - ' + __('Starts an Ethereum simulator (ganache) and runs the tests using the simulator') + '\n' +
+              '                       embark - ' + __('Uses the node associated with an already running embark process') + '\n' + 
+              '                       ' + __('<custom node endpoint> - Connects to a running node available at the end point and uses it to run the tests'), 
+              /^(vm|embark|((ws|https?):\/\/([a-zA-Z0-9_.-]*):?([0-9]*)?))$/i, null)
       .option('-d , --gasDetails', __('When set, will print the gas cost for each contract deploy'))
       .option('--locale [locale]', __('language to use (default: en)'))
       .option('--loglevel [loglevel]', __('level of logging to display') + ' ["error", "warn", "info", "debug", "trace"]', /^(error|warn|info|debug|trace)$/i, 'warn')
       .description(__('run tests'))
       .action(function(file, options) {
+        if(options.node === null){
```

Comment:
what happens if you run `embark test` ? 
Does it stop the process?
The default is supposed to be `vm`